src/pybind/mgr/orchestrator/_interface.py

```diff
@@ -969,7 +969,9 @@ def daemon_action(self, action, daemon_type, daemon_id):
         Perform an action (start/stop/reload) on a daemon.
 
         :param action: one of "start", "stop", "restart", "redeploy", "reconfig"
-        :param name: name of daemon
+        :param daemon_type: type of daemon
+        :param daemon_id: Depending on daemon type it can be numerical value or
+        service name
         :rtype: Completion
```

Comment:
/home/jenkins-build/build/workspace/ceph-pr-docs/src/pybind/mgr/orchestrator/_interface.py:docstring of orchestrator.Orchestrator.daemon_action:9: WARNING: Field list ends without a blank line; unexpected unindent.